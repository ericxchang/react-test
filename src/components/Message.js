import React from 'react';

const Message = (props) => <div className={props.className}>{props.children}</div>
export default Message
