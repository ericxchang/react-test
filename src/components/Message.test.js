import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Message from './Message'

const StyledMessage = styled(Message)`
  color: red;
`

test('it works', () => {
  const tree = renderer.create(<StyledMessage>hello test</StyledMessage>).toJSON()
  expect(tree).toMatchSnapshot()
})