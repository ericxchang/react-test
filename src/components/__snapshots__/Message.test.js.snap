// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  color: red;
}

<div
  className="c0"
>
  hello test
</div>
`;
